active/diversity-inclusion.md

```diff
@@ -0,0 +1,68 @@
+Our primary goal is to increase participation from underrepresented groups, create an inclusive community environment, and promote diverse leadership within the Django project. We also aim to provide guidance to the DSF Board of Directors in line with this mandate. 
+The Working Group will pursue this goal in several ways including:
+
+- Recruiting globally representative Working Group members that will advocate for and provide insights on behalf of their local communities. Our members will, where possible, include representatives from each continent/ region.
+- Discussing policy, initiatives, grant proposals and membership to recommend to the DSF Board of Directors in line with the group’s purpose.
+- Communicating and collecting feedback from the Django community to understand how the DSF can better serve and grow a diverse membership.
+- Measuring, evaluating, and sharing the DSF’s progress towards its diversity initiatives on a regular basis.
+- The Working Group reserves the right to consult with outside experts.
+- Serving as the Djangonaut Space other mentorship programs D&I liaison.
+- Performing Audit or report on D&I of Django events and contributors. 
+- Supporting the organisation of regional Django related events around the world in line with the group's purpose.
+- Providing support to the DSF board of directors in managing the DSF Membership and the DSF members.
+- 
+
+## Initial membership
+
+- Chair: Ariane Djeupang
+- Co-Chair: Keanya Phelps
+- Board Liaison: Afi Maame
+- Other members:
+   - Eli Rosselli
+   - Priya Pahwa
+   - TBD 
+
+## Future membership
+
+- Folks that would like to join the Diversity and Inclusion Work Group should send an email to the WG `diversity-inclusion@djangoproject.com` introducing themselves. The WG will vote on new member requests verbally at meetings, and the board can take part to the vote.
+- The working group adopts the DSF [Code of Conduct](https://www.djangoproject.com/conduct/) .
+- Any actions made against the principles in the Code of Conduct (found by a simple majority of 51%+) will result in the acting person being removed from the working group.
+- Diversity and Inclusion working group voting members are required to confirm their intent to continue participating in the Working Group within one month of a new DSF Board election. This will be led by chair/vice-chair. If there is no response within two weeks, then the individual is automatically removed.
+- A member of the Diversity and Inclusion Working Group, can step down at any time, and the working group will work to replace that voting member.
+- The Chair / Co-Chair will be responsible for coordinating the group, scheduling meetings, renewing the group’s membership, and ensuring that the group’s activities align with its scope and responsibilities.
+- 
+
+## Budget
+
+- TBA
```

Comment:
I’m not sure if the plan is to complete this before or after the group’s initial formation?